### DISCUSSION TITLE
remove the horizontal separator lines ?

### DISCUSSION THREAD

**Discussion by ffred** [XusinboyBekchanov/VisualFBEditor#436] [Q&A] [answered]
Hi,
just discovered VisualFBEditor and trying it right now. that's a really great work, thanks !!
is it possible to remove the horizontal separator lines in the editor ?? I didn't find anything for that in the settings.. (could have missed something ?).
thanks..

**Comment by ffred** [comment]
looked again.. I probably have to change one value in the editor theme, to set this color to the same as the background ?

[Edit]: doesn't seem to be the good way because it's also hiding the rectangular marker...
probably need to set one Scintilla parameter for the folding line. (found doc [here](http://transit.iut2.upmf-grenoble.fr/doc/anjuta/ScintillaDoc.html) in the "folding" section).

**Comment by XusinboyBekchanov** [comment, marked as answer]
Added: Show Horizontal Separator Lines option:
https://github.com/XusinboyBekchanov/VisualFBEditor/commit/3b88850660f89f56a5c62d31d399b245b70b169f

Added this option.

I don't use Scintilla. I have my own Edit Control.

**Comment by ffred** [comment]
ok.. thank you !
